### ISSUE TITLE
influxdb3 doesn't work with next.js 

### ISSUE THREAD

**Issue by seonglae** [InfluxCommunity/influxdb3-js#234]
### Specifications

* Client Version: 0.6.0
* InfluxDB Version: cloud
* Platform: mac & linux

### Code sample to reproduce problem

```js
const client = new InfluxDBClient()
```

### Expected behavior

making new client

### Actual behavior

Error in next.js build

When I try to build this error occurs
```
./node_modules/.pnpm/apache-arrow@15.0.0/node_modules/apache-arrow/builder/valid.mjs
Dynamic Code Evaluation (e. g. 'eval', 'new Function', 'WebAssembly.compile') not allowed in Edge Runtime 
Used by createIsValidFunction
Learn More: https://nextjs.org/docs/messages/edge-dynamic-code-evaluation

Import trace for requested module:
./node_modules/.pnpm/apache-arrow@15.0.0/node_modules/apache-arrow/builder/valid.mjs
./node_modules/.pnpm/apache-arrow@15.0.0/node_modules/apache-arrow/builder.mjs
./node_modules/.pnpm/apache-arrow@15.0.0/node_modules/apache-arrow/Arrow.dom.mjs
./node_modules/.pnpm/@influxdata+influxdb3-client@0.6.0/node_modules/@influxdata/influxdb3-client/dist/index.browser.mjs
```

Even if I allow [dynamic evaluation](https://nextjs.org/docs/messages/edge-dynamic-code-evaluation) then this error occurs in runtime
I think apache-arrow dependency has some relation with this issue 
This is not an edge environment and this is node.js environment. I don't know why browser module is used though I didn't used [`influxdb3-browser`](https://www.npmjs.com/package/@influxdata/influxdb3-client-browser) npm module
```
TypeError: Super constructor null of anonymous class is not a constructor
    at (node_modules/.pnpm/@influxdata+influxdb3-client@0.6.0/node_modules/@influxdata/influxdb3-client/dist/index.browser.mjs:5:31354)
    at (node_modules/.pnpm/@influxdata+influxdb3-client@0.6.0/node_modules/@influxdata/influxdb3-client/dist/index.browser.mjs:5:33248)
    at (webpack/bootstrap:21:0)
    at (node_modules/.pnpm/@texonom+nutils@1.2.12/node_modules/@texonom/nutils/build/index.js:1:13502)
    at (webpack/runtime/jsonp chunk loading:34:0)
    at (api/search-notion:middleware.js:1:17)
```

**Comment by bednar** [comment]
Hello @seonglae,

Firstly, we appreciate your use of our client and taking the time to report the issue you've encountered. Your feedback is crucial for us to maintain and improve the quality of our software.

To help us better understand and address the issue, could you please verify if the previous version of the client, version 0.5.0, functions correctly in your environment? This step will aid us in pinpointing the changes that might have introduced the problem.


Best Regards.

**Comment by seonglae** [comment]
I reproduced same issue in that repository during `next build` in Windows & Mac either

```bash
git clone https://github.com/bednar/next-js-influxdb3
cd next-js-influxdb3
pnpm i
pnpm build
``` 

Error logs
```
> next build

   ▲ Next.js 14.1.0

   Creating an optimized production build ...
 ✓ Compiled successfully
 ✓ Linting and checking validity of types
   Collecting page data  ...TypeError: Super constructor null of anonymous class is not a constructor
```
I suspect the `apache-arrow` dependency issue for this

**Comment by footniko** [comment]
@bednar When you run your code in dev mode (e.g. `yarn dev`) it works, but if you prepare it for a production build (with `yarn build`) it fails:

<details>
<summary>Result</summary>

yarn run v1.21.1
$ next build
   ▲ Next.js 14.1.0

   Creating an optimized production build ...
 ✓ Compiled successfully
 ✓ Linting and checking validity of types    
   Collecting page data  ...TypeError: Super constructor null of anonymous class is not a constructor
    at new <anonymous> (/Users/test/tmp/influx/next-js-influxdb3/.next/server/app/page.js:6:10903)
    at 3192 (/Users/test/tmp/influx/next-js-influxdb3/.next/server/app/page.js:6:10866)
    at Function.t (/Users/test/tmp/influx/next-js-influxdb3/.next/server/webpack-runtime.js:1:128)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async collectGenerateParams (/Users/test/tmp/influx/next-js-influxdb3/node_modules/next/dist/build/utils.js:919:21)
    at async /Users/test/tmp/influx/next-js-influxdb3/node_modules/next/dist/build/utils.js:1138:17
    at async Span.traceAsyncFn (/Users/test/tmp/influx/next-js-influxdb3/node_modules/next/dist/trace/trace.js:151:20)

> Build error occurred
Error: Failed to collect page data for /
    at /Users/test/tmp/influx/next-js-influxdb3/node_modules/next/dist/build/utils.js:1258:15
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  type: 'Error'
}
error Command failed with exit code 1.
</details>

**Comment by bednar** [comment]
@footniko thanks for clarification. I will take a look ASAP

**Comment by footniko** [comment]
It seems to be a webpack issue (https://github.com/webpack/webpack/issues/17711)
Just tried with Next.js pre-released version (14.1.1-canary.45) and it works fine. Looks like v14.0.1 uses a webpack where this issue isn't yet fixed.
